### PR TITLE
removes hexane

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

removes hexane gas from video game

(the ability to hear dchat got removed like half a day ago)

## Why It's Good For The Game

it will speed up atmos a bit with less reactions, the reagent from the gas does the same things as halon + bz and the reaction from it is pretty similar to halon, they are both gases that exist to stop fires, i dont think we need both

## Changelog
:cl:
del: removes hexane
fix: fixes halon's speed thing being removed by consuming tiny amounts of hexane
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -14,13 +14,11 @@ bzformation = 4
...
-hexane_plasma_filtering = 8
 zauker_decomp = 9
 healium_formation = 10
 proto_nitrate_formation = 11
 zauker_formation = 12
 halon_formation = 13
-hexane_formation = 14
 proto_nitrate_response = 15 - 18
 fusion = 19
 metallic_hydrogen = 20
```
Comment (Jared-Fogle) [inline code/modules/atmospherics/gasmixtures/reactions.dm:24]: Move over the priorities so that everything stays sequential.

**Comment by brotherbeyondd** [comment]
Based

File: code/modules/atmospherics/gasmixtures/reactions.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: In a more perfect world priorities would be cascading defines, but we can deal with that later

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 2]
```diff
@@ -923,37 +889,6 @@ nobiliumsuppression = INFINITY
 			air.temperature = max(((temperature * old_heat_capacity - energy_used) / new_heat_capacity), TCMB)
 	return REACTING
 
-/datum/gas_reaction/hexane_plasma_filtering
-	priority = 8
```
Comment (Jared-Fogle) [inline code/modules/atmospherics/gasmixtures/reactions.dm:927]: You can't just change the comment, you need to change all these priority variables of everything above it.

**Comment by ExcessiveUseOfCobblestone** [comment]
@Ghilker

File: code/modules/atmospherics/gasmixtures/reactions.dm (hunk 2 above)
Comment (Fikou) [reply code/modules/atmospherics/gasmixtures/reactions.dm:927]: fuck

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 3]
```diff
@@ -14,14 +14,12 @@ bzformation = 4
...
-proto_nitrate_response = 15 - 18
+zauker_decomp = 8
+healium_formation = 9
+proto_nitrate_formation = 10
+zauker_formation = 11
+halon_formation = 12
+proto_nitrate_response = 13 - 18
 fusion = 19
 metallic_hydrogen = 20
 nobiliumsuppression = INFINITY
```
Comment (LemonInTheDark) [inline code/modules/atmospherics/gasmixtures/reactions.dm:25]: Priority should really be a define at this point, I'll do that tomorrow TM, but in the mean time, you should update up to fusion.

**Review by LemonInTheDark — COMMENTED** [review]
You don't appear to have missed anything major, and I agree with  the concept as of now, clean up the priorities and we should be good.

File: code/modules/atmospherics/gasmixtures/reactions.dm (hunk 3 above)
Comment (Fikou) [reply code/modules/atmospherics/gasmixtures/reactions.dm:25]: i did??

Comment (LemonInTheDark) [reply code/modules/atmospherics/gasmixtures/reactions.dm:25]: Ah you're right, Misread some things

**Comment by LemonInTheDark** [comment]
Ah you should clean the sprite up too, <icons/effects/atmospherics.dmi>

**Pull request merged**